File: cpp/test/test_12.cpp
#include "../src/code_12.cpp"
#include "cpp_deps/boilerplate.hpp"

void test(Solution& sol, const json& input, const json& output) {
    int num = input["num"].get<int>();
    string expected = output.get<string>();
    string result = sol.intToRoman(num);
    CHECK_EQ(result, expected);
}

// asdf
TEST_CASE("") {
    TEST(12);
}

File: python/tests/test_12.py
from python.src.code_12 import Solution
from python.tests.python_deps.boilerplate import BoilerTest


def input_fetcher(case):
    return case["input"]["num"]


def test():
    test = BoilerTest(12, input_fetcher)
    solution = Solution().intToRoman
    test.run_tests(solution)
